packages/commonwealth/server/migrations/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "private": "true"
```

Comment:
Can we not work around this? Very weird to have to add a package.json for something that is a not a package. Especially to just have `private: true` inside.

Reply:
We could rename past migrations to `.cjs` to fix this. That might be something to put in a fast follow PR though (just because of the number of files changed), but we should at least get it on a branch and test it now.

Reply:
Alright makes sense. Let's get a fast-follow ticket up for this.